src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TasksController;
+
+/**
+ * Abstract class Command. All commands will inherit from this class.
+ */
+public abstract class Command {
+
+    /**
+     * An abstract method that every child class needs to implement
+     * @param controller Duke task controller
+     * @param taskText if it's add task command, then pass the context of the task.
+     * @param taskTime if it's add Event or Deadline, then pass the time
+     * @param taskIndex if it's mark or unmark command, then pass the task number
+     * @param keywords if it's find command, then pass the keywords
+     * @param storage Duke IO processor
+     */
```

Comment:
Punctuation could be added behind each parameter description

Reply:
Good point, it seems I didn't add punctuation to most of my comments. I'll pay attention to that next time.